src/core/Akka.TestKit/TestKitExtension.cs

```diff
@@ -37,5 +41,27 @@ public static TestKitSettings For(ActorSystem system)
         {
             return system.GetExtension<TestKitSettings>();
         }
+        
+    }
+
+    public static class EnvelopeExtention
+    {
+        public static async Task<T> TryTakeAsync<T>(this BufferBlock<T> bufferBlock)
```

Comment:
`.ReceiveAsync()` throws an exception if no item was found. This extension method was created to keep things neat and tidy